docs/standard/security/decrypting-data.md

```diff
@@ -22,7 +22,7 @@ Decryption is the reverse operation of encryption. For secret-key encryption, yo
 
 The decryption of data encrypted with symmetric algorithms is similar to the process used to encrypt data with symmetric algorithms. The <xref:System.Security.Cryptography.CryptoStream> class is used with symmetric cryptography classes provided by .NET to decrypt data read from any managed stream object.
 
-The following example illustrates how to create a new instance of the default implementation class for the <xref:System.Security.Cryptography.Aes> algorithm. The instance is used to perform decryption on a <xref:System.Security.Cryptography.CryptoStream> object. This example first creates a new instance of the **Aes** implementation class. Next it creates a **CryptoStream** object and initializes it to the value of a managed stream called `myStream`. Next, the **CreateDecryptor** method from the **Aes** class is passed the same key and IV that was used for encryption and is then passed to the **CryptoStream** constructor.
+The following example illustrates how to create a new instance of the default implementation class for the <xref:System.Security.Cryptography.Aes> algorithm. The instance is used to perform decryption on a <xref:System.Security.Cryptography.CryptoStream> object. This example first creates a new instance of the **Aes** implementation class. Next it reads IV value from a managed stream called `myStream`. Next it creates a **CryptoStream** object and initializes it to the value of the managed stream called `myStream`. Next, the **CreateDecryptor** method from the **Aes** class is passed the read IV and the same key that was used for encryption and are then passed to the **CryptoStream** constructor.
 
 ```vb
```

Comment:
If you're adding VB and C# code, all code examples must have functional equivalents. For example, this VB should have a C# equivalent - otherwise you end up with a mix of the two languages and the language dropdown doesn't work.

Reply:
Am I missing something? The C# code is just below the VB code. Or should C# code be always first?
I didn't change this part and in currently published version both languages work.

Reply:
But this VB snippet is followed by C#.

```vb
Dim aes As Aes = Aes.Create()
Dim cryptStream As New CryptoStream(myStream, aes.CreateDecryptor(key, iv), CryptoStreamMode.Read)
```

```csharp
Aes aes = Aes.Create();
CryptoStream cryptStream = new CryptoStream(myStream, aes.CreateDecryptor(key, iv), CryptoStreamMode.Read);
```

Unless we are talking about different snippet.